pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools"]
+requires = ["setuptools>=61.0"]
```

Comment:
Why 61?

Reply:
That's when pyproject.toml support was added.   https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html

Reply:
I see thanks!